website/docusaurus.config.js

```diff
@@ -3,8 +3,8 @@ const { resolve } = require('path');
 const { linkDocblocks, transpileCodeblocks } = require('remark-typescript-tools');
 
 module.exports = {
-  title: 'RTK Query',
-  tagline: 'An advanced data fetching & caching library built on redux-toolkit',
+  title: 'Powerful data fetching and caching capabilities for every everyone',
```

Comment:
"every everyone"?  not just _some_ everyones? :)

Reply:
I have no idea what I'm doing 😆 . I suck at marketing copy, but I think it's getting better.